Description: Patch nvp settings
Author: arosen <arosen@awesome>
--- quantum-2012.2.nicira.7.1.orig/etc/quantum.conf
+++ quantum-2012.2.nicira.7.1/etc/quantum.conf
@@ -19,7 +19,7 @@ bind_port = 9696
 # api_extensions_path =
 
 # Quantum plugin provider module
-core_plugin = quantum.plugins.sample.SamplePlugin.FakePlugin
+core_plugin = quantum.plugins.nicira.nicira_nvp_plugin.QuantumPlugin.NvpPluginV2
 
 # Paste configuration file
 api_paste_config = api-paste.ini
--- quantum-2012.2.nicira.7.1.orig/etc/dhcp_agent.ini
+++ quantum-2012.2.nicira.7.1/etc/dhcp_agent.ini
@@ -5,7 +5,7 @@
 # Where to store dnsmasq state files.  This directory must be writable by the
 # user executing the agent. The value below is compatible with a default
 # devstack installation.
-state_path = /opt/stack/data
+state_path = /var/lib/quantum
 
 # The DHCP agent will resync its state with Quantum to recover from any
 # transient notification or rpc errors. The interval is number of
@@ -16,7 +16,7 @@ state_path = /opt/stack/data
 # matches you plugin.
 
 # OVS
-interface_driver = quantum.agent.linux.interface.OVSInterfaceDriver
+interface_driver = quantum.agent.linux.interface.OVSVethInterfaceDriver
 # LinuxBridge
 #interface_driver = quantum.agent.linux.interface.BridgeInterfaceDriver
 # Ryu
